config/media_store.sql

```diff
@@ -30,9 +31,6 @@ DROP TABLE IF EXISTS Playlist;
 DROP TABLE IF EXISTS PlaylistTrack;
 DROP TABLE IF EXISTS Track;
 
-DROP ZONE IF EXISTS Chinook;
-DROP ZONE IF EXISTS ChinookReplicated;
-
```

Comment:
Let's keep the DROP ZONE so that running the script twice will apply all changes. For instance, without these lines if I increase the number of replicas to 3 and run the script again, that would _not_ reflected in the cluster.